qcodes/instrument_drivers/Harvard/Decadac.py

```diff
@@ -26,12 +26,14 @@ def _dac_v_to_code(self, volt):
         based on the minimum/maximum values of a given channel.
         Midrange is 32768.
         """
+        if volt < self.min_val or volt >= self.max_val:
```

Comment:
Is this `>=` on purpose? Without looking at the implementation, as a user I would interpret `max_val` as the maximally allowed value....

Reply:
It is on purpose, but I agree that it is not too fitting.

The thing is that `min_val` maps to `0` and `max_val` maps to `2**16`. So that's one value too many. But I think that current technology can solve even this.